ocfweb/atool/constants.py

```diff
@@ -0,0 +1,25 @@
+PASSWORD_ENCRYPTION_PUBKEY = '/etc/ocfweb/create.pub'
+KRB_KEYTAB = '/etc/ocfweb/chpass.keytab'
+
+CMDS_HOST = 'ssh.ocf.berkeley.edu'
+CMDS_HOST_KEYS_FILENAME = '/etc/ocfweb/ssh_known_hosts'
+
+TEST_OCF_ACCOUNTS = (
+    'sanjay',  # an old, sorried account with kerberos princ
+    'alec',  # an old, sorried account with no kerberos princ
+    'guser',  # an account specifically made for testing
+    'nonexist',  # this account does not exist
+)
+
+TESTER_CALNET_UIDS = (
+    '872544',   # daradib
+    '1034192',  # ckuehl
+    '869331',   # tzhu
+)
+
+# comma separated tuples of CalLink OIDs and student group names
+TEST_GROUP_ACCOUNTS = (
+    (91740, 'The Testing Group'),  # needs to have a real OID, so boo
+    (46187, 'Open Computing Facility'),  # good old ocf
+    (46692, 'Awesome Group of Awesome')  # boo another real OID
+)
```

Comment:
nit: this stuff mostly belongs in the settings file (or the config file), but we can deal with it later